dev-aws/customer-billing/topics.tf

```diff
@@ -417,3 +417,31 @@ resource "kafka_topic" "dummy-placeholder" {
     "cleanup.policy"    = "delete"
   }
 }
+
+# this topic is used during the transition from exstream to bex bill
+# generation and contains AccountReadyToBeFulfilledEvents 
+resource "kafka_topic" "transition-exstream-fulfilment" {
+  name               = "transition.exstream-fulfilment"
```

Comment:
I wonder if this should be called `transition-exstream-account-fulfiled`? Or something similar in past tense?

Because fulfilment events are what comes after the Invoice Fulfilment Orchestrator, where this is looking at the account being fulfilled.

Reply:
These topics are temporary though, so I don' think this matters too much XD I have approved.

Reply:
We will start using `.` for topic names. We shouldn't take any The other new topics are called `bex.internal.bill_fulfilled` (https://kafka-ui.prod.uw.systems/ui/clusters/shared-aws/all-topics?perPage=25)

So I could change it to `transition.XXX` something. It doesn't use `bex` because it is on our kafka, so everything is bex.

I could change it to something like: `transition.exstream.fulfilment_request`. What do you think about that?

Reply:
I think topic naming will be important for us, happy to discuss and change

Reply:
`fulfilment_request` sounds good to me 👍

Reply:
Will anything other than the account fulfilled request go through this topic? is it worth including account in the title. I realise it does make it more specific/less generic which usually a bad thing (IMO)

Reply:
Updated the names. 

I don't think any other events will go on that topic, the event might be called AccountReadyToBeFulfilledEvent but I think what we do is generating an invoice. I don't like the event name too much